tests/api_tests.py

```diff
@@ -387,42 +396,41 @@ async def test_user_restricted_access_not_permitted(api_client: APISessionClient
 @pytest.mark.e2e
 @pytest.mark.asyncio
 @pytest.mark.parametrize(
-    'test_product_and_app',
+    "test_product_and_app",
     [
         {
-            'scopes': ['urn:nhsd:apim:user-nhs-login:P6:immunisation-history'],
-            'requested_proofing_level': 'P9',
-            'identity_proofing_level': 'P6'
+            "scopes": ["urn:nhsd:apim:user-nhs-login:P6:immunisation-history"],
+            "requested_proofing_level": "P9",
+            "identity_proofing_level": "P6",
+            "authorised_targets": "FOO,BAR",
         },
         {
-            'scopes': ['urn:nhsd:apim:user-nhs-login:P6:immunisation-history'],
-            'requested_proofing_level': 'P5',
-            'identity_proofing_level': 'P6'
-        }
+            "scopes": ["urn:nhsd:apim:user-nhs-login:P6:immunisation-history"],
+            "requested_proofing_level": "P5",
+            "identity_proofing_level": "P6",
+            "authorised_targets": "FOO,BAR",
```

Comment:
> Will these not fail once the backend is built?

Yeah, fair. On reflection it's a bit of an issue that we're going to have to keep these tests in sync with the backend repo (i.e. if we remove a target from the backend). It would be better if we had a single source of truth, but i'll raise that for discussion elsewhere as it ties into what we were talking about on Friday at the retro regarding integration between the repos